File: Easy/Java/485. Max Consecutive Ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int track = 0;

        for(int i = 0; i < nums.length; i++){
            if(nums[i] == 1)
                track++;
            else
                track = 0;
            count = Math.max(count, track);
        }

        return count;
    }
}

File: Easy/Python/485. Max Consecutive Ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        count = 0
        track = 0

        for i in range(len(nums)):
            if nums[i] == 1:
                track += 1
            else:
                track = 0
            count = max(count, track)

        return count
